parsers/GT.py

```diff
@@ -0,0 +1,137 @@
+# The arrow library is used to handle datetimes
```

Comment:
I can't run this python file unless I add those lines at the top, that will tell my python that the file is utf-8:
```python
#!/usr/bin/python
# -*- coding: utf-8 -*-
```